src/routes/safe/components/Transactions/TxList/TxSummary.tsx

```diff
@@ -51,31 +63,55 @@ export const TxSummary = ({ txDetails }: Props): ReactElement => {
           <TxShareButton safeTxHash={txDetails.detailedExecutionInfo.safeTxHash} />
         </div>
       )}
+      {txData?.operation === Operation.DELEGATE && (
```

Comment:
Be aware that I've adjusted this in the [trusted flag PR](https://github.com/gnosis/safe-react/issues/3099).

Reply:
Should we consider that in this PR or when merging `dev` into the epic branch?

Reply:
It's up to you. It was merged into dev just now